test/test_events.js

```diff
@@ -211,6 +211,26 @@ describe('events', () => {
     });
   });
 
+  it('should emit an event if a job fails to extend lock', done => {
+    const LOCK_RENEW_TIME = 1;
+    queue = utils.buildQueue('queue fails to extend lock', {
+      settings: {
+        lockRenewTime: LOCK_RENEW_TIME
+      }
+    });
+    queue.once('lock-extension-failed', lockingFailedJob => {
```

Comment:
it would be great to also include the err object in this test and check that the error matches the expected error (like the expected error has the same error message).